lib/shrine.rb

```diff
@@ -295,7 +295,9 @@ def extract_filename(io)
         # Attempts to extract the MIME type from the IO object.
         def extract_mime_type(io)
           if io.respond_to?(:content_type) && io.content_type
-            warn "The \"mime_type\" Shrine metadata field will be set from the \"Content-Type\" request header, which might not hold the actual MIME type of the file. It is recommended to load the determine_mime_type plugin which determines MIME type from file content."
+            if opts[:mime_type_security_warning] != :suppress
```

Comment:
I prefer a simpler name like `:mime_type_warning`.

On a higher level, do we want to go down the path of having individual warning control or a global warning control?

Reply:
I can change the name to whatever y'all like. 

I think individual control is needed. I thought about this in the first place because I specifically did _not_ want to disable _all_ warnings (including some I dont' even know about). 

I think the shrine source code probably doesn't want to either -- it does a monkey-patch to disable all warnings, next to a comment saying it was only intending to disable a certain warning. If the shrine tests produced _other_ warnings that were intended to be produced by shrine code (they do not at present), I don't think we'd want em hidden. 

The monkey-patch approach is still available for disabling all warnings, which is all that was available before anyway.

Reply:
This is the only warning there is a previous known use case for disabling -- I just want it to be possible to disable just that warning, which is what I think was desired, and not via a monkey-patch is bonus too.

Reply:
So you prefer individual warning control - ok. Me too, for now.

I only raise the question for us to make a proactive decision on warnings going forward.